ext/pgsql/pgsql.c

```diff
@@ -3082,8 +3083,8 @@ PHP_FUNCTION(pg_copy_to)
 	ExecStatusType status;
 	char *csv = (char *)NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "OP|Ss", &pgsql_link, pgsql_link_ce,
-		&table_name, &pg_delimiter, &pg_null_as, &pg_null_as_len) == FAILURE
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "OP|Sss", &pgsql_link, pgsql_link_ce,
+		&table_name, &pg_delimiter, &pg_escape, &pg_null_as, &pg_null_as_len) == FAILURE
```

Comment:
I think that should rather be:
```suggestion
	if (zend_parse_parameters(ZEND_NUM_ARGS(), "OP|SsS", &pgsql_link, pgsql_link_ce,
		&table_name, &pg_delimiter, &pg_null_as, &pg_null_as_len, &pg_escape) == FAILURE
```